cfme/fixtures/automate.py

```diff
@@ -0,0 +1,32 @@
+import fauxfactory
+import pytest
+
+
+@pytest.fixture(scope='module')
+def domain(appliance):
+    """This fixture used to create automate domain - Datastore/Domain"""
+    domain = appliance.collections.domains.create(
+        name=fauxfactory.gen_alpha(),
+        description=fauxfactory.gen_alpha(),
+        enabled=True)
+    yield domain
+    domain.delete_if_exists()
+
+
+@pytest.fixture(scope="module")
+def namespace(domain):
+    """This fixture used to create automate namespace - Datastore/Domain/Namespace"""
+    yield domain.namespaces.create(
+        name=fauxfactory.gen_alpha(),
+        description=fauxfactory.gen_alpha()
+    )
+
```

Comment:
Q: Why don't you have some cleanup here afterwards?

Reply:
@lina-nikiforova 
- I am agree with you.
- But ```namepaces```, ```classes```, ````methods````, ```instances``` etc. are not require cleanup because these things are completely dependent ```domain``` creation. 
- Without creating domain we can not proceed for these things.
- Still I have added cleanup for these fixtures.